packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -0,0 +1,164 @@
+        for (uint256 i; i < exampleFoundationConfig.owners.length; i++) {
+            _callViaSafe({
+                _safe: Safe(payable(safe)),
+                _target: safe,
+                _data: abi.encodeCall(OwnerManager.addOwnerWithThreshold, (exampleFoundationConfig.owners[i], 1))
+            });
+        }
+        _callViaSafe({
+            _safe: Safe(payable(safe)),
+            _target: safe,
+            _data: abi.encodeCall(
+                OwnerManager.removeOwner, (exampleFoundationConfig.owners[0], msg.sender, exampleFoundationConfig.threshold)
+            )
+        });
+        addr_ = safe;
+        vm.stopBroadcast();
+    }
+
+    /// @notice Deploy a LivenessGuard for use on the Security Council Safe.
+    ///         Note this function does not have the broadcast modifier.
+    function deployLivenessGuard() public returns (address addr_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        addr_ = address(new LivenessGuard(councilSafe));
+
+        save("LivenessGuard", address(addr_));
+        console.log("New LivenessGuard deployed at %s", address(addr_));
+    }
+
+    /// @notice Deploy a LivenessModule for use on the Security Council Safe
+    ///         Note this function does not have the broadcast modifier.
+    function deployLivenessModule() public returns (address addr_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        address guard = mustGetAddress("LivenessGuard");
+        SecurityCouncilConfig memory councilConfig = _getExampleCouncilConfig();
+
+        addr_ = address(
+            new LivenessModule({
+                _safe: councilSafe,
+                _livenessGuard: LivenessGuard(guard),
+                _livenessInterval: councilConfig.livenessInterval,
+                _thresholdPercentage: councilConfig.thresholdPercentage,
+                _minOwners: councilConfig.minOwners,
+                _fallbackOwner: councilConfig.fallbackOwner
+            })
+        );
+
+        save("LivenessModule", address(addr_));
+        console.log("New LivenessModule deployed at %s", address(addr_));
+    }
+
+    /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
+    function deployAndConfigueSecurityCouncilSafe() public returns (address addr_) {
+        Safe safe = Safe(payable(deploySafe("SecurityCouncilSafe")));
+
+        address guard = deployLivenessGuard();
+
+        vm.startBroadcast();
+        _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(GuardManager.setGuard, (guard)) });
+        console.log("LivenessGuard setup on SecurityCouncilSafe");
+
+        SecurityCouncilConfig memory exampleCouncilConfig = _getExampleCouncilConfig();
+        // Add the owners, keeping the threshold at 1 for now.
+        for (uint256 i = 0; i < exampleCouncilConfig.safeConfig.owners.length; i++) {
```

Comment:
Same question around safe owner config, especially given we're using `vm.startBroadcast(address(safe))` anyway

Reply:
OK, resolving this just for tracking purposes. Let's discuss in the other thread.